### PR TITLE
Webhook

### PR DESCRIPTION
**Pull request by KHOIOHK87**

### CODE REVIEW

**Comment by KHOIOHK87** [comment]
Refer to README for the expected behaviors.

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+  - docker
+
+before_install:
+  - if [[ -v DOCKERHUB_TOKEN ]]; then echo "${DOCKERHUB_TOKEN}" | docker login -u "icdevops" --password-stdin; fi
+
+script:
+  # Audit npm packages. Fail build whan a PR audit fails, otherwise report the vulnerability and proceed.
+  - if [ "${TRAVIS_PULL_REQUEST}" != "false" ]; then npx audit-ci --low; else npx audit-ci --low || true; fi
+  - npm run lint
+  - docker build --rm -t "quay.io/razee/impersonation-webhook:${TRAVIS_COMMIT}" .
```
Comment (alewitt2) [inline .travis.yml:14]: do you have a strong preference on `impersonation-webhook`? When i created the quay.io image registry i used `impersonationwebhook`. I can create a new one if you really want, but i was thinking youd use `impersonationwebhook` to match the repo name

```suggestion
  - docker build --rm -t "quay.io/razee/impersonationwebhook:${TRAVIS_COMMIT}" .
```

**Comment by alewitt2** [comment]
looks great! What did you find out about nested razee resources?

File: .travis.yml (hunk 1 above)
Comment (KHOIOHK87) [reply .travis.yml:14]: I just thought that `impersonationwebhook` is a bit hard to read. Long word :) Yeah if you can change the repo name, that would be great! If not, I can update this.

**Comment by alewitt2** [comment]
awesome! that was my suspicion, but glad you were able to verify it!

File: .travis.yml (hunk 1 above)
Comment (alewitt2) [reply .travis.yml:14]: ok updated: https://quay.io/repository/razee/impersonation-webhook?tab=info

**Pull request merged**